workflow-templates/im-build-db-ci.yml

```diff
@@ -134,20 +134,29 @@ jobs:
-          run-tests: true # TODO: Remove this if your database doesn't have tests.
-          nuget-username: ${{ secrets.ARTIFACTORY_USERNAME }} # TODO: If install-mock-db-objects is set to false, this can be removed
-          nuget-password: ${{ secrets.ARTIFACTORY_API_KEY }} # TODO: If install-mock-db-objects is set to false, this can be removed
-          # If you need to run this job on Windows runners, then remove db-username and db-password so integrated security will be utilized
-          db-username: sa
-          db-password: ${{ secrets.ACTIONS_RUNNER_MSSQL_SA_PWD }}
+          migration-files-path: '' # TODO: Add the path to the directory with your migration files
+          # TODO: The parameters above are the bare minimum needed to use the action. Below are listed the optional parameters you may need to use.
+          #       Uncomment the ones you need and fill out the additional TODO's.
+          # db-server-port: 1433 # TODO: This is the default value in the action. Delete this if you don't need to change it.
+          # use-integrated-security: false # TODO: This is the default value in the action. Delete this if you don't need to change it.
+          # db-username: 'sa' # The default system administrator account that comes with SQL Server. This is the only account set up in our self hosted Action Runners' SQL Server instances.
+          # db-password: ${{ secrets.ACTIONS_RUNNER_MSSQL_SA_PWD }} # This is an org level secret. It is the password for the sa account.
+          # install-mock-db-objects: true
+          # mock-db-object-dependency-list: '' # TODO: Add a json list of objects specifying your dependencies. The format is [{"version":"1.0.0","packageName":"dbo.Something","nugetUrl":"https://nuget.pkg.github.com/my-org/my-repo/dbo.Something.nupkg"}]
+          # incremental: false # TODO: This is the default value in the action. Delete this if you don't need to change it.
+          # create-database-file: '' # TODO: Add the path to the create database file (often called Initialization.sql)
+          # run-tests: true
+          # test-files-path: '' # TODO: Add the path to the directory with your test files
+          # test-timeout: '' # TODO: Add the time, in seconds, to wait for all the tests to run before timing out. The default is 300. Delete if 300 is fine or if you aren't running tests.
+          # drop-db-after-build: false # TODO: This is the default value in the action. Delete this if you don't need to change it.
+          # should-validate-migrations: false  # TODO: This is the default value in the action. Delete this if you don't need to change it.
+          # seed-data: true
+          # seed-data-files-path: '' # TODO: Add the path to the directory with your seed data files
+          # managed-schemas: dbo  # TODO: This is the default value in the action. If you need to change it add commas between the schemas (i.e. dbo,OtherSchema), otherwise delete this line.
```

Comment:
What if instead of listing out every arg they could use we put in the most common items and then add a more detailed todo saying something like by default this runs on localhost port 1433 and will run tests and do seed data, etc.  Check out the action (with link) for more details on configuring db setup?

Reply:
That sounds like a good idea! I think that would be less overwhelming for sure.

Reply:
Ok I took a stab at those changes. Let me know what you think!